src/features/CareViewPatientsSummary/components/PatientDetailsCell.jsx

```diff
@@ -0,0 +1,181 @@
+import { FormattedMessage } from "react-intl";
+import propTypes from "prop-types";
+import PropTypes from "prop-types";
+import { CareViewContext } from "../../../context/CareViewContext";
+import { getCurrentShiftTimes } from "../../../utils/DateTimeUtils";
+import WarningIcon from "../../../icons/warning.svg";
+import { getIPDPatientDashboardUrl } from "../../../utils/CommonUtils";
+
+export const PatientDetailsCell = ({
+  patientDetails,
+  bedDetails,
+  careTeamDetails,
+  navHourEpoch,
+  newTreatments,
+  visitDetails,
+}) => {
+  const { person, uuid } = patientDetails;
+  const { ipdConfig, provider, handleRefreshPatientList } =
+    useContext(CareViewContext);
+  const [bookmark, setBookmark] = useState({});
+  const { shiftDetails: shiftConfig = {} } = ipdConfig;
+  const nurse = bookmark?.provider;
+  const formattedNurseName = nurse?.display.includes(" - ")
+    ? nurse?.display.split(" - ")[1]
+    : nurse?.display;
+  const nurseName = nurse && formattedNurseName;
+  const isBookmarked = Object.keys(bookmark).length !== 0;
+
+  const getBookmarkStatus = (careTeamDetailsData) => {
+    const nearestHourEpochInMilliseconds = navHourEpoch.startHourEpoch * 1000;
+    if (!careTeamDetailsData || !careTeamDetailsData.participants) return {};
+
+    for (const participant of careTeamDetailsData.participants) {
+      if (
+        nearestHourEpochInMilliseconds >= participant.startTime &&
+        nearestHourEpochInMilliseconds < participant.endTime
+      ) {
+        return participant;
+      }
+    }
+    return {};
+  };
+  const handleBookmarkClick = async (uuid) => {
+    const { startDateTime, endDateTime } = getCurrentShiftTimes(shiftConfig);
+    const bookmarkData = {
+      providerUuid: provider.uuid,
+      startTime: startDateTime,
+      endTime: endDateTime,
+    };
+
+    const unBookmarkData = {
+      uuid: bookmark.uuid,
+      voided: true,
+    };
+
+    const patientData = {
+      patientUuid: uuid,
+      careTeamParticipantsRequest: [bookmarkData],
+    };
+
+    try {
+      if (isBookmarked) {
+        await bookmarkPatient({
+          ...patientData,
+          careTeamParticipantsRequest: [unBookmarkData],
+        });
+        setBookmark({});
+      } else {
+        const response = await bookmarkPatient(patientData);
+        const participant = getBookmarkStatus(response.data);
+        setBookmark(participant);
+      }
+    } catch (e) {
+      handleRefreshPatientList();
+    }
+  };
+
+  useEffect(() => {
+    const participant = getBookmarkStatus(careTeamDetails);
+    setBookmark(participant);
+  }, []);
+
+  return (
+    <td className={"patient-details-container"}>
+      <div className={"care-view-patient-details"}>
+        <div className={"admission-details"}>
+          <HospitalBed16 />|<span>{bedDetails.bedNumber}</span>|
+          <Link
+            data-testid="identifier-ipd-dashboard"
+            onClick={() =>
+              window.open(
```

Comment:
Can we pass the URL In the href prop rather than window.open?
We can do 
<Link href = "getIPDPatientDashboardUrl(
                  patientDetails.uuid,
                  visitDetails.uuid)" />

Reply:
Ok, Looks like it is a targeted tab. We can ignore this comment.